Source/MLX/Documentation.docc/Organization/arithmetic.md

```diff
@@ -169,7 +170,7 @@ Note: the `-` and `/` operators are not able to be linked here.
 - ``sign(_:stream:)``
 - ``sin(_:stream:)``
 - ``sinh(_:stream:)``
-- ``softMax(_:axes:stream:)``
+- ``softMax(_:axes:precise:stream:)``
```

Comment:
I know it's a bit late, but your comment on naming made me think  that this one should really be `softmax`. It's such a common function that its treated as one word at this point

Reply:
No worries, I can deprecate this spelling and leave an indication for Xcode to suggest `softmax()`.  I presume the NN layer should also be `Softmax`?

Reply:
No worries, I can deprecate this spelling and leave an indication for Xcode to suggest `softmax()`.  I presume the NN layer should also be `Softmax`?

Reply:
Great! Deprecation + suggestion is nice. And yea the `nn.Softmax` is right, that's what is in Python as well.

Reply:
How about the other "soft" names?  `SoftPlus`, `SoftSign`, `LogSoftMax` (and related functions)?  Same treatment?

Reply:
`hardSwish` -> `hardswish`?

Reply:
OK, I did all the Soft* ones and can circle back on hardSwish

Reply:
Sounds good! `LogSoftmax` definitely. The other ones are less common / less standard so either way seems ok. What you have now is consistent with `mlx.nn` so that seems good though.